GVFS/GVFS.FunctionalTests/FileSystemRunners/BashRunner.cs

```diff
@@ -233,7 +239,14 @@ public override void DeleteFile_FileShouldNotBeFound(string path)
 
         public override void DeleteFile_AccessShouldBeDenied(string path)
         {
-            this.DeleteFile(path).ShouldContain(permissionDeniedMessage);
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))

```

Comment:
If we start getting more of these, we could create a `WindowsBashRunner` and a `MacBashRunner`. This seems fine for now though.